### ISSUE TITLE
demos not working

### ISSUE THREAD

**Issue by ManuelHaas** [yjs/yjs#502]
Is there a problem with the demos linked on the website? None of them (quill, monaco, ...) is working for me:
![image](https://user-images.githubusercontent.com/10881695/217008466-2266d490-39b6-4ecb-8f88-9e41925e1666.png)

**Comment by davidkircos** [comment]
yeah none of them work :(

**Comment by burakakca** [comment]
![image](https://github.com/yjs/yjs/assets/56919004/621b9c74-8c14-4b9b-bcb6-6a73b89a7b22)

The servers are not standing up

**Comment by genki** [comment]
It is still not working. Is this project alive?

**Comment by kalagin** [comment]
@dmonad still not working

**Comment by KesavarajaRK** [comment]
Update us please.

**Comment by dmonad** [comment]
Yeah, I'm sorry for that. Currently, there are too many people using the public webrtc signaling servers.  Anything I setup is immediately overloaded. Suggestions are welcome.

**Comment by KesavarajaRK** [comment]
Sorry to hear that, but we can implement it like TLDRAW has done. Whitelist a domain and allow the demo only from that domain.
